### PR TITLE
Switching Color from RGBA to HSV

### PR DESCRIPTION
**Pull request by cr**

### CODE REVIEW

File: src/values.rs [hunk 1]
```diff
@@ -680,48 +680,42 @@ impl PartialOrd for Temperature {
 pub enum Color {
     /// # JSON
     ///
-    /// Values are represented as an object {r: float, f: float, b: float, a: float},
-    /// where each color is between 0 and 1. Field `a` may be omitted, in which case
-    /// it is taken to be 0.
+    /// Values are represented as an object {h: float, s: float, v: float},
+    /// where s and v are between 0 and 1.
```
Comment (Yoric) [inline src/values.rs]: What's the range for `h`? Is any number acceptable?

**edit** Ah, I see below that it's mod 360. Can you mention that in the doc?

File: src/values.rs [hunk 2]
```diff
@@ -680,48 +680,42 @@ impl PartialOrd for Temperature {
...
-    /// assert_eq!(a, 0.8);
+    /// let h = serialized.find("h").unwrap().as_f64().unwrap();
+    /// assert_eq!(h, 220.5);
+    /// let s = serialized.find("s").unwrap().as_f64().unwrap();
+    /// assert_eq!(s, 0.8);
+    /// let v = serialized.find("v").unwrap().as_f64().unwrap();
+    /// assert_eq!(v, 0.4);
     ///
     ///
     /// println!("Testing parsing error (value not in [0, 1])");
```
Comment (Yoric) [inline src/values.rs]: Could you take the opportunity to replace the word `value` with `saturation`?

**Comment by Yoric** [comment]
r=me with trivial docchanges

File: src/values.rs (hunk 2 above)
Comment (cr) [inline src/values.rs]: Done.

File: src/values.rs (hunk 1 above)
Comment (cr) [inline src/values.rs]: What do you mean by doc? I have mentioned it in several places in this file.

Comment (cr) [inline src/values.rs]: I added a line right there.

Comment (Yoric) [inline src/values.rs]: Yeah, that's what I meant. Thanks.

**Pull request merged**